java/maven/src/test/java/net/sf/mmm/code/java/maven/impl/MavenBridgeImplTest.java

```diff
@@ -48,6 +53,24 @@ public void testReadModel() {
         DependencyHelper.create("org.apache.maven", "maven-core", "${maven.version}"));
   }
 
+  /**
+   * Testing if a valid child pom artifact dependency version can be resolved from its parent using dependencyManagement
+   * (mvn help:effective-pom still works)
+   */
+  @Test
+  public void testResolveDependencyManagementVersionFromParent() {
+
+    // given
+    File mavenProjectDirectory = new File(rootTestPath, "localmavenproject/maven.project/core"); // test Maven project
+
+    // when
+    MavenBridgeImpl reader = new MavenBridgeImpl();
+    Model model = reader.readEffectiveModel(new File(mavenProjectDirectory, "pom.xml"));
+
+    // then
+    assertThat(model).isNotNull();
```

Comment:
shouldnt this test assert something more reasonable in addition?
However, surely agreed that the POM shall be parsable and therefore the model shall not be null. If that failked before sufficient for the PR to be acceptable :)